File: plumbot-velocity/src/main/java/me/regadpole/plumbot/tool/StringTool.java
package me.regadpole.plumbot.tool;

import java.util.regex.Matcher;
import java.util.regex.Pattern;

public class StringTool {

    public static String filterColor(String text){

        String regEx = "§[0-9a-zA-Z]";
        Pattern p = Pattern.compile(regEx);
        Matcher matcher = p.matcher(text);
        return matcher.replaceAll("").trim();

    }
}

File: common/src/main/kotlin/me/regadpole/plumbot/tool/StringTool.kt
package me.regadpole.plumbot.tool

import java.util.regex.Matcher
import java.util.regex.Pattern

object StringTool {
    fun filterColor(text: String?): String {
        val regEx = "§[0-9a-zA-Z]"
        val p: Pattern = Pattern.compile(regEx)
        val matcher: Matcher = p.matcher(text)
        return matcher.replaceAll("").trim()
    }
}